samples/SimpleDataSource/Tables/TableBase.cs

```diff
@@ -0,0 +1,39 @@
+﻿using Microsoft.Performance.SDK;
+using Microsoft.Performance.SDK.Processing;
+using System;
+using System.Collections.Generic;
+
+namespace SampleCustomDataSource.Tables
+{
+    //
+    // This is a sample base class for all regular and metadata tables in your project which helps simplify management of them.
+    // 
+    // A table is a logical collection of similar data points. 
+    // Things like CPU Usage, Thread Lifetimes, File I/O, etc. are all tables
```

Comment:
This sentence doesn't make much sense on a literal level. File I/O is not a table, it's an action of moving data. Maybe this sentence should end with "can be mapped to tables" instead?